src/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureSVN.java

```diff
@@ -49,6 +50,24 @@
 public class SourceCodeDisclosureSVN extends AbstractAppPlugin {
 
 	/**
+	* if we got a 404 or a redirect specifically, then this is NOT a match
+	* note that since we are simply relying on the file existing or not, we 
+	* will not attempt any fuzzy matching. Old school.
+	* Checks based on this are necessary, otherwise a recursive scan on nodes 
+	* in the url path cause lots of false positives.
+	* 	MOVED_PERMANENTLY - 301
+	* 	FOUND - 302
+	* 	SEE_OTHER - 303
+	* 	NOT_MODIFIED - 304
+	* 	USE_PROXY - 305 (306 is currently unused)
+	* 	TEMPORARY_REDIRECT - 307
+	* 	NOT_FOUND - 404
+	*/
+	private static final int[] UNWANTED_RESPONSE_CODES= {301,302,303,304,305,307,404};
```

Comment:
This can be converted directly to a List:
`List<Integer> UNWANTED_RESPONSE_CODES = Arrays.asList(301, 302, 303, 304, 305, 307, 404);`
the other conversions can be deleted.